### PR TITLE
Some Claw buffs

### PR DESCRIPTION
**Pull request by EgorDinamit**
## About The Pull Request

- Buffs some of the Claw attacks, namely the dash(it will no longer be going in just one direction).
- Decreases cooldowns for some abilities.
- Adds continuous dash attack: It just calls the swift dash 5 times in a row.
- Changes proc names to CamelCase since that's what we're doing nowadays.
- Additionally made all abilities be called by invoke async.

## Why It's Good For The Game

- The easiest to dodge attack got buffed, that's about it. Claw was rather weak for super-late game boss.
- Ditto.
- Makes it easier for the Claw to chase someone who's running too fast.
- Code improvement/standartization.
- Same here.

### CODE REVIEW

File: ModularTegustation/tegu_mobs/the_claw.dm [hunk 1]
```diff
@@ -11,7 +11,7 @@
 	icon_living = "claw"
 	icon_dead = "claw_dead"
 	icon = 'ModularTegustation/Teguicons/48x48.dmi'
-	faction = list("Head")
+	faction = list("hostile", "Head")
```
Comment (dsmith328) [inline ModularTegustation/tegu_mobs/the_claw.dm:14]: Why? Not necessarily a negatively charged question just wondering the intent here

**Comment by dsmith328** [comment]
`blink` causes runtime errors should it's target no longer exist.

File: ModularTegustation/tegu_mobs/the_claw.dm (hunk 1 above)
Comment (EgorDinamit) [reply ModularTegustation/tegu_mobs/the_claw.dm:14]: I don't want Claw to be fighting random abnormalities. It is here to fight the players, not random things.

**Comment by EgorDinamit** [comment]
Updated.

**Review by Kitsunemitsu — APPROVED** [review]
Should be fine from a code and balance perspective. Code looks good and all LAZYLEN runtimes should be caught.
Mostly just number and proc name change.

**Pull request merged**